### ISSUE TITLE
post image for individual posts isn't constrained

### ISSUE THREAD

**Issue by noctividus** [cnb/News_Manager-getsimplecms#265]
When a post has a featured image, that image is not constrained to any size. Therefore, it over-runs its div.

Example: http://richmayfield.com/offthegrid/post/social-pool

**Comment by cnb** [comment]
I don't get it. I can't find any featured image in your example post.

**Comment by cnb** [comment]
You can set a custom post image size for the single post view with custom settings, examples:
```
single imagesize 400 300
```
or if you want the original image in single posts:
```
single imagesize full
```

**Comment by noctividus** [comment]
Thanks for the quick reply. I removed that featured image, but the same happens with an unconstrained image. Like, if you insert an image, then it just makes itself whatever size. For my application, I added the following to my theme's main CSS. However, this may be some CSS that could be adapted for your plugin to make it work a little better. 
`/*News Manager  Additions*/

/* This style makes the images correctly scaled on the main blog page */
.nm_post_image img {
    max-width: 60%;
}
/* This style makes un-dimensioned images stay a good size in posts */
.nm_post_content img {
	max-width: 80%;
}`
